cmd/podmanV2/images/history.go

```diff
@@ -60,11 +64,50 @@ func history(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	row := "{{slice $x.ID 0 12}}\t{{toRFC3339 $x.Created}}\t{{ellipsis $x.CreatedBy 45}}\t{{$x.Size}}\t{{$x.Comment}}\n"
-	if cmdFlags.Human {
-		row = "{{slice $x.ID 0 12}}\t{{toHumanDuration $x.Created}}\t{{ellipsis $x.CreatedBy 45}}\t{{toHumanSize $x.Size}}\t{{$x.Comment}}\n"
+	if opts.format == "json" {
```

Comment:
lint apparently wants a switch here instead.

Reply:
Not sure if that's more readable, but lint is happy now.